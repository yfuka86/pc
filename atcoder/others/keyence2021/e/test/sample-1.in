7
4 3 1 2 1000 2000 3000
