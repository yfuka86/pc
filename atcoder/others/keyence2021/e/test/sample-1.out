6004
6004
6004
6001
5007
4007
4007
4007
